File: src/Front/app/src/main/java/com/example/front/model/DTO/RatingDTO.kt
package com.example.front.model.DTO

import com.google.gson.annotations.SerializedName

data class RatingDTO (
    @SerializedName("raterId") val raterId:Int,
    @SerializedName("ratedId") val ratedId:Int,
    @SerializedName("communication") val communication:Float,
    @SerializedName("reliability") val reliability:Float,
    @SerializedName("overallExperience") val overallExperience:Float,
    @SerializedName("average") val average:Float,
    @SerializedName("rater") val rater:UserDTO,
    @SerializedName("rated") val rated:UserDTO,
)

File: src/back/BLL/Dtos/RatingDto.cs
namespace back.BLL.Dtos
{
    public class RatingDto
    {
        public int RaterId { get; set; }
        public int RatedId { get; set; }
        public float Communication {  get; set; }
        public float Reliability { get; set; }
        public float OverallExperience { get; set; }
    }
}
